packages/story-editor/src/components/panels/design/border/borderWidth.js

```diff
@@ -21,62 +21,27 @@ import PropTypes from 'prop-types';
 import { useCallback } from '@web-stories-wp/react';
```

Comment:
@Trapsta I just realized that this PR doesn't add the eyedropper to the Border color input. Just adding `hasEyedropper` here should do it (but test it out just in case):
https://github.com/google/web-stories-wp/blob/306146c63488ee3d20365a980feeebb2c6067558/packages/story-editor/src/components/panels/design/border/border.js#L72-L79